### PR TITLE
- add action press to move page manage call

### PR DESCRIPTION
**Pull request by tatthangcntt**
It does not display multiple incoming calls after brekeke phone try to pickup the call just canceled.
**Issue**:
1. Make 3 incoming calls to brekeke phone following oders: 101, 102, 103
2. the last one 103 rings and displays on top of the brekeke phone's screen.
3. 103 cancels and the brekeke phone try to pickup the call at the same time.
=> The call is disconnected, But it shows a screen like
current list -> 102 is ringing
background list -> 101 is riniging
4. The brekeke phone try to select arrow button to move to main screen of the phone.
=> Actual: Now, it just shows only one call (102) on Call bar, and cannot move change back to background list anymore.
Expected: It can change back the screen to see background list.
5. Press Disconnected, 
=> Actual: the call 102 is disconnected. It does not display remaing call (101), while 101 keep dialing status.

### CODE REVIEW

**Comment by namnm** [comment]
Trong code hiện tại mới fix dc 4. Còn cái 5. chưa fix

File: src/components/CallNotify.tsx [hunk 1]
```diff
@@ -79,10 +80,13 @@ export const CallNotify = observer(() => {
     <Wrapper>
       {callStore.shouldRingInNotify(c.callkeepUuid) && <IncomingItem />}
       <View style={css.Notify}>
```
Comment (namnm) [inline src/components/CallNotify.tsx:82]: Cho TouchableOpacity ra ngoài này dc k để vùng touch nó rộng ra?

**Review by namnm — CHANGES_REQUESTED** [review]

**Comment by tatthangcntt** [comment]
> Trong code hiện tại mới fix dc 4. Còn cái 5. chưa fix

Hiện tại chưa tái hiện được cái số 5!

File: src/components/CallNotify.tsx (hunk 1 above)
Comment (tatthangcntt) [reply src/components/CallNotify.tsx]: Ok!

**Pull request merged**